packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -305,6 +305,8 @@ export class DbMutationQueue {
      * by the server. All MutationBatches in this queue with batchIds less
      * than or equal to this value are considered to have been acknowledged by
      * the server.
+     *
+     * NOTE: this is deprecated and no longer used by the code.
```

Comment:
I'm not sure if I can remove it from here, given that we can't remove `lastAcknowledgedBatchId` from the database schema?

Reply:
Yeah, if for no other reason I think we need to keep it because it's used by the migration that deletes all of the acknowledged mutations.